__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const { topicData, userData, articleData, commentData } = require('../db/data/test-data/')
+const seed = require('../db/seeds/seed');
+const request = require ('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const { forEach } = require('../db/data/test-data/topics');
+
+beforeEach(() => {
+    return seed({topicData, userData, articleData, commentData})
+})
+
+afterAll(() =>{
+    db.end()
+})
+
+describe('GET /api/topics' , () =>{
+    it('responds with an array of topic objects with correct properties' , () =>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) =>{
+            console.log(body.topics)
+            expect(Array.isArray(body.topics)).toBe(true)
+
+            body.topics.forEach((obj) => expect(obj).toHaveProperty('slug'))
```

Comment:
When we are using a forEach in our test envrioment we should check the length of the array we are getting back. If an empty array was to be passed through here his test would still pass.
Is there a need to have 2 seperate forEach's here? could we not have multiple expects within the one forEach?